### PR TITLE
Display the amount of time for Migration completion

### PR DESCRIPTION
**Pull request by AparnaKarve**
Three noticeable changes in the PR -
- Displays the amount of time for Migration completion - calculated as a difference of `delivered_on` and `created_on`
- Sorting was updated to place the most recent plan at the top of the list.
(In the screenshot below, `MyPlanToday` was created today - April 13th)
- ~For time < 1 day, display the time in `h:m:s` format (previously, it did not show seconds)~
Display time in `hh:mm:ss` format consistently (for time < 1 day or otherwise) 

screen shot 2018-04-13 at 1 56 53 pm

---------
Shows a row with Time in Days HH:MM:SS
screen shot 2018-04-13 at 2 01 35 pm

Fixes https://github.com/priley86/miq_v2v_ui_plugin/issues/194

### CODE REVIEW

File: app/javascript/react/screens/App/Overview/components/Cards/MigrationsCompletedCard/MigrationCompletedRow.js [hunk 1]
```diff
@@ -33,7 +34,7 @@ const MigrationCompletedRow = ({ migration }) => {
   } else if (days === 1) {
     elapsedTime = sprintf(__('1 day %s:%s'), hours, minutes);
   } else {
-    elapsedTime = sprintf(__('%s:%s'), hours, minutes);
+    elapsedTime = sprintf(__('%s:%s:%s'), hours, minutes, seconds);
```
Comment (serenamarie125) [inline app/javascript/react/screens/App/Overview/components/Cards/MigrationsCompletedCard/MigrationCompletedRow.js]: @AparnaKarve it is possible to do hh:mm:ss for alignment purposes?

Comment (AparnaKarve) [reply app/javascript/react/screens/App/Overview/components/Cards/MigrationsCompletedCard/MigrationCompletedRow.js]: @serenamarie125 Sure. I have made the required changes and have updated the screenshot above.

Comment (serenamarie125) [reply app/javascript/react/screens/App/Overview/components/Cards/MigrationsCompletedCard/MigrationCompletedRow.js]: great thank you @AparnaKarve !

**Comment by priley86** [comment]
this looks great! I've mimicked this design in #212 for now...

merging.

**Pull request merged**